e2e-tests/cypress/fixtures/application.js

```diff
@@ -75,8 +83,23 @@ const application = {
     [FINAL_DESTINATION]: 'DZA',
   },
   EXPORTER_COMPANY: {
+    [COMPANY_NUMBER]: COMPANIES_HOUSE_NUMBER,
+    [COMPANY_NAME]: 'DHG PROPERTY FINANCE LIMITED',
+    [COMPANY_INCORPORATED]: '10 April 2014',
+    [COMPANY_ADDRESS]: {
+      addressLine1: 'Unit 3 Lewis Court',
+      addressLine2: 'Portmanmoor Road',
+      careOf: '',
+      locality: 'Cardiff',
+      region: 'South Glamorgan',
+      postalCode: 'CF24 5HQ',
```

Comment:
Can we use UKEF address?

Reply:
address for the company, has to be exact